### PR TITLE
add example "managing_imports"

### PR DESCRIPTION
**Pull request by RandalSchwartz**
This completes #76

### CODE REVIEW

**Review by SandroMaglione — CHANGES_REQUESTED** [review]
This look great! Just some revisions on the `README`.

Also feel free to add a link to the example in the [`Examples`](https://github.com/SandroMaglione/fpdart#-examples) section of the main fpdart `README`.

File: example/managing_imports/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Managing Imports
+
+Naming things is hard.  Sometimes, the same name gets used for different things. In Dart, naming conflicts can be mitigated through the use of import prefixes, as well as show and hide operations.
```
[the remark this answers is not in the document]
Comment (RandalSchwartz) [reply example/managing_imports/README.md:3]: I thought the first paragraph *was* a summary.  Can you give a suggestion of what you're looking for?

File: example/managing_imports/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Suppose you decide to use `fpdart` with your Flutter program. You'll quickly discover that `fpdart` uses `State` as a class name, which conflicts with the `State` class in Flutter.
+
+That's problem 1.
+
+Now also suppose you also choose to use `fpdart`'s `Tuple2` class. That's a lot less likely to conflict with anything, but it's still possible. However, you also decide you need a `Tuple3`. `fpdart` doesn't have one. (Yet.)
+
+However, you found one in the `tuple` package, along with `Tuple4` and `Tuple5`, even though it doesn't have much more than element accessors. Close enough for your application.
```
[the remark this answers is not in the document]
Comment (RandalSchwartz) [reply example/managing_imports/README.md:11]: added

**Comment by RandalSchwartz** [comment]
So do I need to do anything to turn it back to you?

File: README.md [hunk 3]
```diff
@@ -28,29 +27,32 @@ Fpdart is inspired by [fp-ts](https://gcanti.github.io/fp-ts/), [cats](https://t
...
-- [💡 Motivation](#-motivation)
-  - [Goal](#goal)
-  - [Comparison with `dartz`](#comparison-with-dartz)
-- [🤔 Roadmap](#-roadmap)
-- [📃 Versioning](#-versioning)
-- [😀 Support](#-support)
-- [👀 License](#-license)
-
-
+- [`Fpdart`](#fpdart)
```
Comment (SandroMaglione) [inline README.md:30]: Could you remove this and indent the list of paragraphs without it (see current `README`?

**Review by SandroMaglione — CHANGES_REQUESTED** [review]
Just one last change and we are good to go.

Thanks for the fixes 👍

File: README.md (hunk 3 above)
Comment (RandalSchwartz) [reply README.md:30]: I don't know what "this" means here.  I don't understand the red/green diff and how it applies to README.

Comment (RandalSchwartz) [reply README.md]: OK, I think I got it.  Just removing from README.md for now.  Add it back if you want.

**Review by SandroMaglione — APPROVED** [review]

**Comment by RandalSchwartz** [comment]
How is the test failing?  Something I put in my branch?

**Comment by SandroMaglione** [comment]
I upgraded the dependencies and now everything works fine.

Thanks again @RandalSchwartz 🙏

**Pull request merged**